### PR TITLE
Use the 'agent' term more often

### PR DESCRIPTION
**Pull request by daniel-beck**
Also remove two untranslated files.

### Proposed changelog entries

* Too minor

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [n/a] (If applicable) Jira issue is well described
- [n/a] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by StefanSpieker — APPROVED** [review]

File: core/src/main/java/hudson/slaves/SlaveComputer.java [hunk 1]
```diff
@@ -975,7 +975,7 @@ public String getOSDescription() throws IOException, InterruptedException {
     private static final class SlaveVersion extends MasterToSlaveCallable<String,IOException> {
         public String call() throws IOException {
             try { return Launcher.VERSION; }
-            catch (Throwable ex) { return "< 1.335"; } // Older slave.jar won't have VERSION
+            catch (Throwable ex) { return "< 1.335"; } // Older agent.jar won't have VERSION
```
Comment (timja) [inline core/src/main/java/hudson/slaves/SlaveComputer.java:978]: isn't it still called slave.jar or was that renamed?

**Review by timja — APPROVED** [review]
one q, but thanks!

File: core/src/main/java/hudson/slaves/SlaveComputer.java (hunk 1 above)
Comment (daniel-beck) [reply core/src/main/java/hudson/slaves/SlaveComputer.java:978]: Both names/URLs exist, this is the preferred one.

https://github.com/jenkinsci/jenkins/blob/62546443947fafecc94a5e3b47bb9129bed8e148/core/src/main/java/hudson/model/Slave.java#L732

**Review by jvz — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**